decision_service/dsclientlib/model_mgmt/ds_model_mgmt.h

```diff
@@ -0,0 +1,47 @@
+#pragma once
+#include "../utility/ds_object_factory.h"
+
+// Declare const pointer for internal linkage  
```

Comment:
The semantics of this is unclear to me---I think I need a walkthrough.

Reply:
Good explanation here.
[https://stackoverflow.com/a/998457](https://stackoverflow.com/a/998457)

---
In reply to: [187115294](https://github.com/JohnLangford/vowpal_wabbit/pull/1479#discussion_r187115294) [](ancestors = 187115294)